app/test/unit/infer-comparison-branch-test.ts

```diff
@@ -46,7 +54,7 @@ function createTestGhRepo(
         ? defaultBranch.split('/')[1]
         : defaultBranch
     }`,
-    `${name.indexOf('/') !== -1 ? name.split('/')[1] : name}.git`,
```

Comment:
@iAmWillShepherd I updated this test because the clone URLs originally set did not contain a host. They're all passing again, but let me know if you're still attached to this behaviour.